L/libblastrampoline/bundled/cmake/yggdrasilenv.cmake

```diff
@@ -0,0 +1,54 @@
+# Environment setup for libblastrampoline to allow the CMake FindBLAS and FindLAPACK
+# scripts work with this library in the Yggdrasil environment.
+
+set( BB_NBITS  $ENV{nbits} )
+set( BB_TARGET $ENV{target} )
+set( BB_DLLEXT $ENV{dlext} )
+set( BB_LIBDIR $ENV{libdir} )
+set( BB_INCDIR "$ENV{prefix}/include" )
```

Comment:
```suggestion
set( BB_INCDIR "$ENV{includedir}" )
```

Reply:
Ah, that is the variable for that. I had tried `${incdir}` at first, and it didn't work, so I just went with this since it was the next things I remembered,